From 5e2c3049f8e6c9c7aa20c00111af1d354c9f1da9 Mon Sep 17 00:00:00 2001
From: Dmitry Tantsur <dtantsur@redhat.com>
Date: Tue, 17 Mar 2015 09:52:35 +0100
Subject: [PATCH] Remove pbr dependency

---
 ironicclient/__init__.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ironicclient/__init__.py b/ironicclient/__init__.py
index e01624c..7460981 100644
--- a/ironicclient/__init__.py
+++ b/ironicclient/__init__.py
@@ -15,13 +15,12 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import pbr.version
 
 from ironicclient import client
 from ironicclient import exc as exceptions
 
 
-__version__ = pbr.version.VersionInfo('python-ironicclient').version_string()
+__version__ = "REDHATIRONICCLIENTVERSION"
 
 __all__ = [
     'client',
